micro-rdk/src/common/analog.rs

```diff
@@ -35,12 +35,26 @@ impl AnalogReader<u16> for FakeAnalogReader {
     fn read(&mut self) -> Result<u16, Self::Error> {
         self.internal_read()
     }
+    fn resolution(&self) -> AnalogResolution {
+        Default::default()
+    }
+}
+
+#[derive(Debug, Default)]
```

Comment:
do we want to be returning derived Default values here? We'd always be returning a `min_range`, `max_range`, and `step_size` of `0.0`. 

Would it not be better to return some "realistic" defaults that we can test the FakeAnalogReader against later?

Reply:
yeah, sure I could throw in some numbers I guess